rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -208,6 +191,11 @@ def get_world_pose_and_speed(self, msg):
         pose_gps = Pose.rospose_to_pose(current_rospose)
         return World.gps_to_world_pose(pose_gps), current_speed
 
+    def local_controller_process(self):
```

Comment:
might be better if they are named local_controller_thread and path_planner_thread, since multi-processing is a different kind of paralleism than multi-threading.

Reply:
Will fix, I actuallly tried process first lol

Reply:
@Jackack  Fixed